README.md

```diff
@@ -1,65 +1,643 @@
+
+###### context/source/standalone/
+
+- `Dockerfile.debian.local`, Debian-based container image manifest for Pretalx; continuation of the old `pretalx/standalone` image
+- `Dockerfile.debian.remote`, Debian-based container image manifest for Pretalx; extended with supervisor; used for the `pretalx/standalone` image
+- `supervisord.conf`, configuration to run multiple unprivileged processes in the same container
+
+###### context/source/standalone.extended.cron/
+
+- `crontab`, configuration to run periodic tasks; adapted to be run via the supervisor
+- `Dockerfile.debian.remote`, Debian-based container image manifest for Pretalx, extended with cron and plugins
+- `supervisord.conf`, configuration to run multiple privileged and unprivileged processes in the same container; adapted for use with cron
+
+##### context/standalone/
+
+The standalone context is provided for compatibility with the `pretalx/standalone` image created with the `pretalx-docker` repository before 05.2024. These containers run both processes of the main Gunicorn process and the Celery task worker. This is useful for testing, but they cannot be scaled independently in a real-world scenario, why migration to the newer `pretalx/pretalx` image with running both processes in separate containers is recommended.
+
+This depreciates the old standalone container, which was tightly coupled with source code in a git submodule, which has been depreciated. A new version that offers a migration path with a decoupled git repository is available in `context/source/standalone/Dockerfile.debian.local`.
+
+###### context/standalone/default/
+
+- `Dockerfile.debian`, Debian-based container image manifest for Pretalx
+- `supervisord.conf`, configuration to run multiple unprivileged processes in the same container
+
+###### context/standalone/extended/
+
+- `Dockerfile.debian`, Debian-based container image manifest for Pretalx, extended with plugins
+
+###### context/standalone/extended.cron/
+
+- `crontab`, configuration to run periodic tasks; adapted to be run via the supervisor
+- `Dockerfile.debian`, container image manifest that adds plugins to Pretalx, extended with cron and plugins
+- `supervisord.conf`, configuration to run multiple privileged and unprivileged processes in the same container; adapted for use with cron
+
+#### legacy/
+
+- `docker-compose.build.yml.example`, overlay example to build an `pretalx-extended` image from the `extended` context
+- `docker-compose.env.example`, Compose, Traefik and Postgres environment example
+- `docker-compose.env.pretalx.example`, Pretalx environment example
+- `docker-compose.yml.example`, Docker Compose manifest example
+
+Please read on thorugh the following sections to find out how they interact with each other.
+
+</details>
+
+## Installation
+
+This repository follows the Pretalx installation instructions as closely as possible.
+
+- [Installation — pretalx documentation](https://docs.pretalx.org/administrator/installation/)
```

Comment:
this seems kind of misleading, since the official docs don't have anything to do with docker at all..